xtable-api/src/main/java/org/apache/xtable/MaturityLevel.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+package org.apache.xtable;
+
+public enum MaturityLevel {
+  /**
+   * New APIs start out in this state. Although enough thought will be given to avoid breaking
+   * changes to the API in the future, sometimes it might need to change based on feedback.
+   */
+  EVOLVING,
```

Comment:
Another option is to make these annotations themselves similar to what Delta does

Reply:
They only have an Evolving one from what I could find, having stable and deprecated in the same class may be better. 

https://github.com/delta-io/delta/blob/master/kernel/kernel-api/src/main/java/io/delta/kernel/annotation/Evolving.java

Reply:
Java already has deprecated and you can still add another class for `Stable`